### ISSUE TITLE
Unsupported error when trying to view the excel files in excel

### ISSUE THREAD

**Issue by huguesfcadets** [cadets-ca/ets#53]
Asked to @kirvanp the file and version of Excel and OS.

**Comment by kirvanp** [comment]
So, on the iOS version of Excel the file just doesn't open. On the Mac, it says the file may be corrupt, but if you click past that everything is fine. I will send you the file directly since gitHub doesn't want it. 

<img width="532" alt="Screen Shot 2020-01-15 at 22 07 54" src="https://user-images.githubusercontent.com/50355421/72495382-c591ec00-37e3-11ea-8dbb-acc964c7631c.png">

**Comment by kirvanp** [comment]
This file.

PK

**Comment by huguesfcadets** [comment]
Is opening the Excel report from an iPad/iPhone a regular usecase?

**Comment by kirvanp** [comment]
Not sure. Another issue with the excel files is that the numbers are stored as text so you can’t Apple formulas to them. For example, if you click an empty cell and try to sum some of the flights, you cannot. This can be bypassed by cutting the numbers and pasting them back in. 

PK

**Comment by huguesfcadets** [comment]
Number as Text is another issue. It can be fixed if we want to do it now. In fact the code exit to create number cells, not used yet as this would require change to the code the generate the HTML/PDF report. Just need prioritization.

**Comment by kirvanp** [comment]
Sounds good. Were you able to take a look at the error I was getting trying to build and run on my iPad?

**Comment by huguesfcadets** [comment]
No. I don't see what is missing. Could you create a new issue for that and give more details.

Hugues Ferland

Gestionnaire – Soutien des applications, Centre de gestion des services TI (CGS TI), Groupe de soutien national aux cadets et aux Rangers juniors canadiens.
Défense nationale / Gouvernement du Canada
hugues.ferland@cadets.gc.ca<mailto:hugues.ferland@cadets.gc.ca> / Tél. : 450-358-7099 X 7105 / RCCC: 668-7105

Manager – Application Support, IT Service Management Centre (IT SMC), National Cadet and Junior Canadian Rangers Support Group
National Defence / Government of Canada
hugues.ferland@cadets.gc.ca<mailto:hugues.ferland@cadets.gc.ca> / Tel: 450-358-7099 X 7105 / CSN: 668-7105

De : kirvanp [mailto:notifications@github.com]
Envoyé : 20 janvier 2020 11:46
À : cadets-ca/ets <ets@noreply.github.com>
Cc : Ferland civ H <Hugues.Ferland@cadets.gc.ca>; Author <author@noreply.github.com>
Objet : Re: [cadets-ca/ets] Unsupported error when trying to view the excel files in excel (#53)

Sounds good. Were you able to take a look at the error I was getting trying to build and run on my iPad?

**Comment by kirvanp** [comment]
Nevermind, I forgot that you fixed that already. My iPad works fine. 

PK
